### DISCUSSION TITLE
Add more vehicle type?

### DISCUSSION THREAD

**Discussion by koopa990** [Joe91/fun-bots#296] [Ideas]
I am confused on why we have chopper and jet, but no standard types such as Jeep\transportation, boat, etc...

**Comment by Firjens** [comment]
Because no one has made a pull request yet with these features.

**Reply by Joe91** [reply]
It's only necessary if they are handled differently in the code

**Reply by koopa990** [reply]
> It's only necessary if they are handled differently in the code

don't you think you should add more than 10? I have a feeling maps like armored shield and nebandan flat have abundance of those armored wheel jeep

**Reply by Joe91** [reply]
There is no need to, if they can behave in code the same like an other vehicle-type. 
I don't think there is much sense in discussing a single enum list without going throught the whole code that uses this enum...
But anyone is welcome to add new functionallity. And once this person is doing so and finds out he or she needs a new type, that won't hold back anyone...
